dice-game-ui-angular/src/app/game/game.component.ts

```diff
@@ -47,7 +47,6 @@ export class GameComponent implements OnInit {
     } catch {
       throw new Error('The game was not started.');
     }
-
```

Comment:
Why did you remove it? It was nice.

Reply:
What actually did I remove? I couldn't find.

Reply:
Empty string.

Reply:
Returned